README.md

```diff
@@ -54,11 +54,18 @@ optional arguments:
                         Enable ARM SCMI support (default: no)
   --ENABLE_WIFI {yes,no}
                         Allow wifi in domd (default: no)
-  --ENABLE_CAN {yes,no}
-                        Allow CAN in domd (default: no)
+  --ENABLE_CAN {mcp2515,seeed_fd,no}
```

Comment:
seeed_fd. Possible typo

Reply:
I don't think seeed_fd is apropriate name since as can be seen from the link:https://www.rcscomponents.kiev.ua/seeed%20technology%20co.,%20ltd/page=15
it producing different modules

Reply:
checked with the link provided in readme: https://wiki.seeedstudio.com/2-Channel-CAN-BUS-FD-Shield-for-Raspberry-Pi/
And still can't understand why seeed_fd is 2-Channel CAN BUS FD Shield with MCP251XFD?
What if I make my own hat with the same chip? or find similar from other vendor? should I create new dtbo?

Reply:
And why first option supports all hats with mcp2515 and second support only specific hat from specific vendor?
Sorry. it's NAK from me